### DISCUSSION TITLE
Allow setting properties and fields with null conditional operator

### DISCUSSION THREAD

**Discussion by JustNrik** [dotnet/csharplang#4275] [Language Ideas] [answered]
For some reason, this isn't allowed:

```cs
someNullableInstance?.SomeProperty = SomeValue;
```

I don't know why this doesn't compile, forcing the user to make a Set method or add more lines of code just to assign a value to a property when the instance isn't null.

Proposal: Allow it, properties are nothing but autogenerated methods, there's no reason why this isn't allowed.

**Comment by svick** [comment, marked as answer]
There is already a championed issue about this: https://github.com/dotnet/csharplang/issues/2883.

**Reply by JustNrik** [reply]
Oh lol, I was searching for this to avoid posting a duplicated issue and I couldn't find it

**Comment by CyrusNajmabadi** [comment]
This is definitely a dupe of #2883
